src/modules/shortcut_guide/shortcut_guide.h

```diff
@@ -68,4 +68,16 @@ class OverlayWindow : public PowertoyModuleIface
             { L"dark", IDS_SETTING_DESCRIPTION_THEME_DARK }
         };
     } theme;
+
+    struct DisplayOnMethod
+    {
+        PCWSTR name = L"display on method";
+        std::wstring value = L"monitor";

```

Comment:
`value` should be one of the three possible values listed from line 78 to 79, I would say we should use "main_display" as default.